clients/ctl/admin-ui/src/features/config-wizard/PrivacyDeclarationForm.tsx

```diff
@@ -76,25 +72,17 @@ const PrivacyDeclarationForm = ({
   const [updateSystem] = useUpdateSystemMutation();
   const [isLoading, setIsLoading] = useState(false);
 
-  const { data: dataCategories } = useGetAllDataCategoriesQuery();
-  const { data: dataSubjects } = useGetAllDataSubjectsQuery();
-  const { data: dataQualifiers } = useGetAllDataQualifiersQuery();
-  const { data: dataUses } = useGetAllDataUsesQuery();
+  // Query subscriptions:
```

Comment:
Note that we still need the hook calls to subscribe to the endpoint, we just don't need to manually store the results in the state. For resources that are used all over the app, we could move the subscription closer to the root of the app and/or [increase cache time](https://redux-toolkit.js.org/rtk-query/usage/cache-behavior#cache-lifetime--subscription-example) .